modules/standard/Path.chpl

```diff
@@ -62,18 +62,78 @@
 */
 module Path {
 
+
 use SysError;
 use Sys;
 
-/* Represents generally the current directory.  This starts as the directory
+
+/*
+   Represents generally the current directory. This starts as the directory
    where the program is being executed from.
- */
+*/
 const curDir = ".";
 /* Represents generally the parent directory. */
 const parentDir = "..";
 /* Denotes the separator between a directory and its child. */
 const pathSep = "/";
 
+
+/*
+   Collapse paths such as `foo//bar`, `foo/bar/`, `foo/./bar`, and
```

Comment:
Maybe include a description of what this function does beyond the example, e.g. "Normalizes a path by eliminating redundant separators and up-level references."

Reply:
Ok. I'll be honest, I just wholesale copied the description from the issue that @lydia-duncan opened.